src/frontend/src/app/services/problems/problems.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { filter, map } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ProblemsService {
+
+  url = "http://localhost:3000"
```

Comment:
Ideally we'd want these to be environment variables so that it's easy to swap them out for production.

We can address this in a separate issue later on though.